queries.sql

```diff
@@ -7,3 +7,39 @@ SELECT name,escape_attempts FROM animals WHERE weight_kg >10.5;
 SELECT * FROM animals WHERE neutered IS true;
 SELECT * FROM animals WHERE name !='Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 and 17.3;
+
+-- day 2
+-- Rename column inside a transaction
+BEGIN TRANSACTION;
+ALTER TABLE animals
+RENAME COLUMN species
+TO unspecified;
+ROLLBACK TRANSACTION;
+
+-- Write species name with a condition inside a  transaction
+BEGIN WORK;
+UPDATE animals SET species='Digimon' WHERE name Like'%mon%';
+UPDATE animals SET species='Pokemon' WHERE species IS NULL;
+COMMIT WORK;
+
+-- delete data and revert
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
```

Comment:
- [ ] Great job so far after the ROLLBACK check if the change was undone, usually run a select statement.